src/foundry/common/data/data.d.mts

```diff
@@ -180,7 +180,7 @@ declare class LightData extends DataModel<LightData.Schema, LightData.Parent> {
    */
   static override LOCALIZATION_PREFIXES: string[];
 
-  static migrateData(source: AnyObject): AnyObject;
```

Comment:
Can you check every usage of `migrateData` and make sure it uses `AnyMutableObject`?